ui/components/ui/box/box.js

```diff
@@ -54,9 +56,12 @@ export const MultipleBackgroundColors = PropTypes.oneOfType([
 ]);
 
 const ArrayOfValidTextColors = PropTypes.arrayOf(ValidTextColors);
+const ArrayOfValidIconColors = PropTypes.arrayOf(ValidIconColors);
 export const MultipleTextColors = PropTypes.oneOfType([
   ValidTextColors,
   ArrayOfValidTextColors,
+  ValidIconColors,
+  ArrayOfValidIconColors,
 ]);
```

Comment:
I'm not sure how to handle this because `color` is used for icons as well. I suppose it's ok because we filter these out in `TEXT` component